src/MagnumPlugins/WebPImporter/WebPImporter.h

```diff
@@ -0,0 +1,147 @@
+#ifndef Magnum_Trade_WebPImporter_h
+#define Magnum_Trade_WebPImporter_h
+/*
+    This file is part of Magnum.
+
+    Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019,
+                2020, 2021, 2022 Vladimír Vondruš <mosra@centrum.cz>
+
+    Permission is hereby granted, free of charge, to any person obtaining a
+    copy of this software and associated documentation files (the "Software"),
+    to deal in the Software without restriction, including without limitation
+    the rights to use, copy, modify, merge, publish, distribute, sublicense,
+    and/or sell copies of the Software, and to permit persons to whom the
+    Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included
+    in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+    DEALINGS IN THE SOFTWARE.
+*/
+
+/** @file
+ * @brief Class @ref Magnum::Trade::WebPImporter
+ */
+
+#include <Corrade/Containers/Array.h>
+#include <Magnum/Trade/AbstractImporter.h>
+
+#include "MagnumPlugins/WebPImporter/configure.h"
+
+#ifndef DOXYGEN_GENERATING_OUTPUT
+#ifndef MAGNUM_WEBPIMPORTER_BUILD_STATIC
+    #ifdef WebPImporter_EXPORTS
+        #define MAGNUM_WEBPIMPORTER_EXPORT CORRADE_VISIBILITY_EXPORT
+    #else
+        #define MAGNUM_WEBPIMPORTER_EXPORT CORRADE_VISIBILITY_IMPORT
+    #endif
+#else
+    #define MAGNUM_WEBPIMPORTER_EXPORT CORRADE_VISIBILITY_STATIC
+#endif
+#define MAGNUM_WEBPIMPORTER_LOCAL CORRADE_VISIBILITY_LOCAL
+#else
+#define MAGNUM_WEBPIMPORTER_EXPORT
+#define MAGNUM_WEBPIMPORTER_LOCAL
+#endif
+
+namespace Magnum { namespace Trade {
+
+/**
+@brief WebP importer plugin
+
+
+@m_class{m-block m-success}
+
+@thirdparty This plugin makes use of the
+    [libPNG](http://www.libpng.org/pub/png/libpng.html) library, released under
```

Comment:
Be sure to eventually update this documentation, it's still referring to libPNG and png.